clash-prelude/src/Clash/Promoted/Nat/Literals.hs

```diff
@@ -3,7 +3,7 @@ Copyright  :  (C) 2013-2016, University of Twente
 License    :  BSD2 (see the file LICENSE)
 Maintainer :  Christiaan Baaij <christiaan.baaij@gmail.com>
 
-Predefined 'SNat' singleton literals in the range [0 .. 1024]
+Predefined 'Clash.Promotoed.Nat.SNat' singleton literals in the range [0 .. 1024]
```

Comment:
```suggestion
Predefined 'Clash.Promoted.Nat.SNat' singleton literals in the range [0 .. 1024]
```

Reply:
Ooh.. weird that the checks didn't catch that. I wonder what's up with that.

Reply:
Hehehe, what you should know about using fully qualified names and haddock: when using fully qualified names it will just assume that the name exists and generates a link for it! Fun fun.